io-engine/src/bin/io-engine-client/v1/stats_cli.rs

```diff
@@ -24,6 +24,13 @@ pub fn subcommands() -> Command {
             .help("Volume target/nexus name"),
     );
 
+    let replica = Command::new("replica").about("Get Replica IO Stats").arg(
+        Arg::new("name")
```

Comment:
Usually the replica CLI is based on replica uuids, why have we decided to use name here?

Reply:
you can have replica with name replica1 and uuid b4865f3d-5c20-454d-949b-23369fdf94e9

Reply:
Right,

For this change, ListStatsOptions has name field,  Since we return IoStats also wiith stats.name . So thought of naming the cli arg as name.

Reply:
Probably we should have both uuid and name as arguments, but as different PR as we'd have to change the api, wdyt @dsharma-dc  @abhilashshetty04 ?